Automattic/wp-calypso
# A4A: Creating a contact support hash linked to the contact form button, allowing it to open when using the URL

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/304

## Proposed Changes

* This PR adds a #contact-support hash, and makes sure that it is set when opening the modal. It also then checks to see if that hash is added to the URL and will open the contact support modal if so.

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/overview`.

* On the overview page, click on the Contact Support button. The support form modal should open, and the URL should have `#contact-support` appended to it. Click to close the modal - the modal should close and `#contact-support` removed from the URL.
* Visit the URL directly, in a new tab, for example `http://agencies.localhost:3000/overview#contact-support`. The modal should open immediately.
* Check that the `calypso_a4a_overview_contact_support_click` event fires once in each case, when clicking the Contact Support button.

## Pre-merge Checklist
